mysql/datadog_checks/mysql/mysql.py

```diff
@@ -399,6 +408,57 @@ def _collect_replication_metrics(self, db, results, above_560):
         results.update(self._get_replica_status(db, above_560, nonblocking))
         return REPLICA_VARS
 
+    def _collect_group_replica_metrics(self, db, results):
+        try:
+            with closing(db.cursor()) as cursor:
+                cursor.execute(SQL_GROUP_REPLICATION_MEMBER)
+                replica_results = cursor.fetchone()
+                status = self.CRITICAL
```

Comment:
```suggestion
                status = self.OK
```
nit for readability/style: it's strange to set status as `critical` before encountering an error, could we update this logic?